app/assets/javascripts/Rickshaw.Graph.DragZoom.js

```diff
@@ -0,0 +1,131 @@
+Rickshaw.namespace('Rickshaw.Graph.DragZoom');
```

Comment:
Note: I need to get some time with an iPad to get the touch swipe functionality working.

@juliusv are there any company ipads on the VPN (so they can access promdash)? it feels SO natural to look at a dashboard graph on an ipad, and swipe to zoom in. I would love to see if there's interest in adoption.